.github/workflows/run_test.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10', 3.11]
+        python-version: [3.8, 3.9, '3.10', 3.11]
```

Comment:
this should be fine as we have test coverage running over 3.7 w/o PDM already, plus PDM now is 3.8+ actually.